source/upgrade.txt

```diff
@@ -70,6 +70,12 @@ Version 6.x Breaking Changes
   ``tlsCertificateKeyFile`` connection options when you call the
   ``MongoClient.connect()`` method, not when you create the
   ``MongoClient`` instance.
+- The major version number of ``mongodb-client-encryption`` must now be the
+  same as the major version number of the {+driver-short+}. For example,
+  {+driver-short+} v6.x.x requires ``mongodb-client-encryption`` v6.x.x.
+- Automatic Encryption methods are now in the {+driver-short+}. You must now
+  import these methods from the driver rather than from
```

Comment:
Suggestion:

I think the first sentence establishes that there's a change, so I would omit the second "now". 

 think "rather than" implies something is preferred less which conflicts with "must".

```suggestion
- Automatic Encryption methods are now in the {+driver-short+}. You must
  import these methods from the driver instead of from
```